keenlabs/KeenClient-Python
# Feature/issue 137 update docs and contributing guidelines

## What does this PR do? How does it affect users?

- Created a new `CONTRIBUTING.md` and `CHANGELOG.md` in project root
- Created templates for filing PRs and issues in `.github`
- Updated `README.md` to point to new resources
- Doesn't affect library users, but improves documentation organization for future project contributors.

## Related tickets?

- Fixes #137 

## How should this be tested?

_This section does not apply to this code because no code changes were made, but I left it intact to show how Github is tightly integrated with the idea of including checkboxes in PRs so you can see how it looks from the "Pull Requests" Tab._

Step through the code line by line. Things to keep in mind as you review:
 - Are there any edge cases not covered by this code?
 - Does this code follow conventions (naming, formatting, modularization, etc) where applicable?

Fetch the branch and/or deploy to staging to test the following:
